man/systemd.exec.xml

```diff
@@ -1745,6 +1745,21 @@ BindReadOnlyPaths=/var/lib/systemd</programlisting>
         <xi:include href="system-or-user-ns.xml" xpointer="singular"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MemoryKSM=</varname></term>
+
+        <listitem><para>Takes a boolean argument. When set, it enables KSM (kernel samepage merging) for
+        the processes. KSM is a memory-saving de-duplication feature. Anonymous memory pages with identical
+        content can be replaced by a single write-protected page. This feature should only be enabled for
+        jobs that share the same security domain. For details, see
+        <ulink url="https://docs.kernel.org/admin-guide/mm/ksm.html">Kernel Samepage Merging</ulink> in the
+        kernel documentation.</para>
+
+        <para>Note that the implementation of this setting might not be possible (for example if KSM is not
+        available or <function>prctl()</function> does not yet support enabling KSM at the process level).</para>
```

Comment:
Hmm, the phrase "the implementation of this setting might not be possible" sounds a bit confusing.

IIUC, maybe something like

"Note that this functionality might not be available, for example if KSM is disabled in kernel, or the kernel doesn't support controlling KSM at the process level through <function>prctl()</function>"

would be better?

Reply:
This explains the limitation better, i made the above change.